.github/workflows/pull_request.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       matrix:
         include:
           - target: x86_64-unknown-linux-musl
-            runs-on: public-amd64-small
+            runs-on: public-amd64-2xlarge
```

Comment:
using a larger machine helps speed up tests and builds quite a lot. I would love if we could have more chunky mac and ARM linux machines